grid/studies/sc/files/grid-studies-2-1.scd

```diff
@@ -1,18 +1,16 @@
 (
 Server.default = Server.local;
 
-~m = MonoM.new("/monome", 0);
+~m = MonomeGrid.new("/128", 0);
```

Comment:
some of these studies using the "128" prefix, others just "monome" ?

Reply:
woop, great catch! simplified to `"/grid"` with a note in the study text about how that's changeable (within the `"/<string>"` format).